src/main/java/com/salesforce/mirus/KafkaMonitor.java

```diff
@@ -89,6 +96,51 @@
         config.getEnablePartitionMatching()
             ? SourcePartitionValidator.MatchingStrategy.PARTITION
             : SourcePartitionValidator.MatchingStrategy.TOPIC;
+    this.routers = this.validateTransformations(config.transformations());
+    this.topicNamePrefix = config.getDestinationTopicNamePrefix();
+    this.topicNameSuffix = config.getDestinationTopicNameSuffix();
+  }
+
+  private List<Transformation<SourceRecord>> validateTransformations(
+      List<Transformation<SourceRecord>> transformations) {
+    List<Transformation<SourceRecord>> regexRouters = new ArrayList<>();
+    for (Transformation<SourceRecord> transform : transformations) {
+      String transformName = transform.getClass().getName();
+      if (transform instanceof RegexRouter) {
+        regexRouters.add(transform);
+        // Slightly awkward check to see if any other routing transforms are configured
```

Comment:
This make me unhappy, but nothing better springs to mind. We will need to be clear about the behaviour in the documentation.  We should update the JavaDoc for KafkaMonitor to explain, and maybe a note in the Readme.